tiledb/__init__.py

```diff
@@ -15,16 +15,16 @@
 # On Windows and whl builds, we may have a shared library already linked, or
 # adjacent to, the cython .pyd shared object. In this case, we can import directly
 # from .libtiledb
-try:
-    import tiledb
-    from .libtiledb import Ctx
-except:
-    try:
-        lib_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), "native")
-        ctypes.CDLL(os.path.join(lib_dir, lib_name))
-    except OSError as e:
-        # Otherwise try loading by name only.
-        ctypes.CDLL(lib_name)
+#try:
+#    import tiledb
+#    from .libtiledb import Ctx
+#except:
+#    try:
+#        lib_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), "native")
+#        ctypes.CDLL(os.path.join(lib_dir, lib_name))
+#    except OSError as e:
+#        # Otherwise try loading by name only.
+#        ctypes.CDLL(lib_name)
```

Comment:
this errors out with a static build. probably need to wrap it in another try block.